lms/djangoapps/verify_student/views.py

```diff
@@ -1006,6 +1007,21 @@ def results_callback(request):
         log.debug("Approving verification for %s", receipt_id)
         attempt.approve()
         status = "approved"
+        checkpoints = VerificationCheckpoint.objects.filter(photo_verification=attempt).all()
+        if checkpoints:
+            course_key = checkpoints[0].course_id
+            credit_requirement = get_credit_requirement(
+                course_key, "reverification", checkpoints[0].checkpoint_name
```

Comment:
`checkpoint_name` attribute will be renamed to `checkpoint_location` so you have to update this attribute and also use `xblock = modulestore.get_item(checkpoints [0].checkpoint_location)` to get the reverification xblock and then use the xblock to get the namespace by using the `xblock.get_credit_requirement_namespace()` to get the namespace instead of using hardcoded "reverification".